DxDispatch/src/dxdispatch/Executor.cpp

```diff
@@ -137,6 +137,20 @@ void Executor::operator()(const Model::DispatchCommand& command)
         return;
     }
 
+    ID3D12Resource* m_timestampResultBuffers;
```

Comment:
nit: only use `m_` in member variable names. This should be something like `timestampResultBuffer`.